termui/show_bug.go

```diff
@@ -643,37 +662,46 @@ func (sb *showBug) addLabel(g *gocui.Gui, v *gocui.View) error {
 	return nil
 }
 
-func (sb *showBug) removeLabel(g *gocui.Gui, v *gocui.View) error {
-	c := ui.inputPopup.Activate("Remove labels")
+func (sb *showBug) edit(g *gocui.Gui, v *gocui.View) error {
+	snap := sb.bug.Snapshot()
 
-	go func() {
-		input := <-c
+	if sb.isOnSide {
+		return sb.editLabels(g, v, snap)
+	}
 
-		labels := strings.FieldsFunc(input, func(r rune) bool {
-			return r == ' ' || r == ','
-		})
+	op, err := snap.SearchTimelineItem(git.Hash(sb.selected))
+	if err != nil {
+		ui.msgPopup.Activate(msgPopupErrorTitle, "Selected field is not editable.")
```

Comment:
This should not be a popup but a panic. You can only have an error here if something is wrong in the code.